src/server.coffee

```diff
@@ -14,6 +17,7 @@ config.alerts = config.get('alerts')
 config.polling = config.get('polling')
 config.reports = config.get('reports')
 config.auditing = config.get('auditing')
+config.agenda = config.get('agenda')
 
 mongoose = require "mongoose"
 exports.connectionDefault = connectionDefault = mongoose.createConnection(config.mongo.url)
```

Comment:
Hey @devcritter, it only possible to remove these if we move all `/model/*` requires out of the master as these exported connection [are](https://github.com/jembi/openhim-core-js/blob/master/src/model/channels.coffee#L79) [used](https://github.com/jembi/openhim-core-js/blob/master/src/model/transactions.coffee#L55-L57) [there](https://github.com/jembi/openhim-core-js/blob/master/src/model/audits.coffee#L60). I'm not too keen to make that change as its rather large, I don't know if it will affect anything else and I don't think we'll save much by doing so. What do you think?